modules/components/stories/AdvancedSqonBuilder/mocks.js

```diff
@@ -0,0 +1,110 @@
+export const sqons = [
```

Comment:
each object in this array follows the sqon spec: https://github.com/overture-stack/arranger/blob/master/modules/middleware/README.md